src/employee/MoEmployeeWorkflows/MoEmployeeCreate.vue

```diff
@@ -101,7 +101,14 @@ export default {
       if (this.formValid) {
         let vm = this
         this.isLoading = true
-        let create = [].concat(this.engagement, this.address, this.association, this.role, this.itSystem, this.manager)
+        let create = [].concat(this.address, this.association, this.role, this.itSystem, this.manager)
+
+        create.forEach(e => {
+          if (!e.validity) {
+            e.validity = this.engagement.validity
+          }
+        })
+        create.push(this.engagement)
```

Comment:
I removed `engagement` from the list to keep it separate from the check. It would still have worked if I had kept it here of course, because it always has a validity set on it.